09-mapping.Rmd

```diff
@@ -754,7 +790,7 @@ Table \@ref(tab:map-pkgs) shows a range of mapping packages are available, and t
 Of note is **cartography**, which generates a range of unusual maps including choropleth, 'proportional symbol' and 'flow' maps, each of which is documented in the vignette [`cartography`](https://cran.r-project.org/web/packages/cartography/vignettes/cartography.html).
 
 ## Exercises
-
+<!-- Exercises are still missing -->
```

Comment:
Exercises are still missing, just a reminder.

Reply:
:+1:

Reply:
True, a priority for the chapter now it's (gradually) coming together. Ideas welcome.